test/support/wallets/wallet_factory.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Mintacoin.WalletFactory do
+  @moduledoc """
+  Allow the creation of wallets while testing.
+  """
+
+  alias Ecto.UUID
+  alias Mintacoin.{Account, Blockchain, Wallet}
+
+  defmacro __using__(_opts) do
+    quote do
+      @spec wallet_factory(attrs :: map()) :: Wallet.t()
+      def wallet_factory(attrs) do
+        %Account{id: default_account_id} = insert(:account)
+        %Blockchain{id: default_blockchain_id} = insert(:blockchain)
+
+        default_public_key = "UE5FYDVAPAEGYYMPJOY4Q54WB53YNX7XU6HIUDWTDYGO3WVDVY4Q"
+
+        default_encrypted_sk =
+          "kLex5P1DpGrseFVQ2UR0lCS5To2AwM3slWQWjtU/R51o545Re2FZgV6lQJtBk+5/ScZDuAkMFYk2mtfP9VTjd4ThwvBxqyH5skG3jsw5Acw"
+
+        default_sk = "72ILTELXNGKM5Y23A74P3B67LJBHRQE4GOP43XIKETOFZQGNTEKA"
+
+        account_id = Map.get(attrs, :account_id, default_account_id)
+        blockchain_id = Map.get(attrs, :blockchain_id, default_blockchain_id)
+        public_key = Map.get(attrs, :public_key, default_public_key)
+        encrypted_secret_key = Map.get(attrs, :encrypted_secret_key, default_encrypted_sk)
+        secret_key = Map.get(attrs, :secret_key, default_sk)
+
+        %Wallet{
+          id: UUID.generate(),
+          public_key: public_key,
+          encrypted_secret_key: encrypted_secret_key,
+          secret_key: secret_key,
+          account_id: account_id,
+          blockchain_id: blockchain_id
```

Comment:
@jhonatan-kmt The factory allow to directly set an account or a blockchain here:
```suggestion
        account = Map.get(attrs, :account, insert(:account))
        blockchain = Map.get(attrs, :blockchain, insert(:blockchain))
        public_key = Map.get(attrs, :public_key, default_public_key)
        encrypted_secret_key = Map.get(attrs, :encrypted_secret_key, default_encrypted_sk)
        secret_key = Map.get(attrs, :secret_key, default_sk)

        %Wallet{
          id: UUID.generate(),
          public_key: public_key,
          encrypted_secret_key: encrypted_secret_key,
          secret_key: secret_key,
          account: account,
          blockchain: blockchain
```
Please review it and tell me what you think.

Reply:
This approach works, but I think it can be a bit confusing, but, if you want, we cant implement it

Reply:
@jhonatan-kmt I think its a better approach because when we are testing for example we can do this:

```elixir
account = insert(:account)
insert(:wallet, account: account)
```
Instead of this:
```elixir
%{id: account_id} = account = insert(:account)
insert(:wallet, account_id: account_id)
```
It's less tricky to set the whole account in the wallet than only the id, what do you think?

Reply:
@CristhianRodriguezMolina Ok, this looks good to me